### PR TITLE
ScheduleApp: add default activities

### PR DESCRIPTION
**Pull request by TomaszLukawskiSam**
[Issue] https://github.com/Samsung/TAU/issues/1181
[Problem]
 - Implement following changes to ScheduleApp
  (1) Set 2 to 3 schedule items as a default status.
  (2) Use ‘navigate up (back key)’ button to ‘Reset’ the application.

[Solution]
  Feature have been implemented

Signed-off-by: Tomasz Lukawski <t.lukawski@samsung.com>

### CODE REVIEW

File: examples/mobile/ScheduleApp/js/app.js [hunk 1]
```diff
@@ -4,30 +4,47 @@
 			this._store = [];
 			this._ui = {}
 		},
+		tempDate = new Date(),
+		defaultStore = [{
+			on: tempDate.getTime(),
+			off: tempDate.getTime() + 3800 * 1000, // + 1h
```
Comment (HunseopJeong) [inline examples/mobile/ScheduleApp/js/app.js]: Isn't it 3600 in an hour?

**Comment by HunseopJeong** [comment]
(2) Use ‘navigate up (back key)’ button to ‘Reset’ the application.
Is it for easy testing?

File: examples/mobile/ScheduleApp/js/app.js [hunk 2]
```diff
@@ -178,11 +199,19 @@
...
+			case "tauback":
+				self._onBack(event);
+				break;
 		}
 	};
 
 	prototype.init = function () {
 		document.addEventListener("pagebeforeshow", this, true);
+		window.addEventListener("tizenhwkey", this, false);
+		document.addEventListener("tauback", this, false);
```
Comment (DohyungLim) [inline examples/mobile/ScheduleApp/js/app.js]: Why are there two features to go back?
Is it a different function?

**Review by DohyungLim — COMMENTED** [review]
1 comment.

File: examples/mobile/ScheduleApp/js/app.js (hunk 2 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/app.js]: The `tizenhwkey` is not the same as `back`. This event not exists on android os. This is very specific case for refresh the app state on `back` button so I had to made solution in non standard way.

**Comment by TomaszLukawskiSam** [comment]
> (2) Use ‘navigate up (back key)’ button to ‘Reset’ the application.
> Is it for easy testing?


Yes. This is for testing. This is feature from issue: https://github.com/Samsung/TAU/issues/1181

File: examples/mobile/ScheduleApp/js/app.js (hunk 1 above)
Comment (TomaszLukawskiSam) [reply examples/mobile/ScheduleApp/js/app.js]: Thanks. I modified. I didn't want a full hour to see the difference more clearly

**Review by HunseopJeong — APPROVED** [review]
Looks good to me

**Review by lslachciak — APPROVED** [review]
OK, but I created new issue for back effect on the add schedule screen https://github.com/Samsung/TAU/issues/1190

**Pull request merged**